src/gis/supported_formats.md

```diff
@@ -0,0 +1,218 @@
+|----------------|-----------------------------------|-------------------------------|-----------------------------------------------------------------------|
+| GeoPackage     | :heart_eyes:                      | :heart_eyes:                  | Collaborative editing via <GitHubRepo id="lutraconsulting/geodiff" /> |
+| Shapefile      | :warning:                         | :warning:                     | Not supported by <GitHubRepo id="lutraconsulting/geodiff" />          |
+| Delimited text | :warning:                         | :warning:                     | Not supported by <GitHubRepo id="lutraconsulting/geodiff" />          | 
+| Virtual layer  | :warning:                         | :warning:                     | Not supported by <GitHubRepo id="lutraconsulting/geodiff" />          |                 
+| PostGIS        | :thumbsup:                        | :thumbsup:                    | Requires internet connection                                          |
+| WFS            | :thumbsup:                        | :thumbsup:                    | Requires internet connection                                          |
+
+::: tip
+Read [more](../manage/synchronisation.md) how the synchronisation in Mergin Maps works.
+:::
+
+
+## Raster data 
+
+| Format         | <MobileAppName /> (Android, iOS)  | QGIS (Windows, macOS, Linux)  | Note                                |   
+|----------------|-----------------------------------|-------------------------------|-------------------------------------|
+| GeoTIFF        | :thumbsup:                        | :thumbsup:                    | png, jpg and webp compression       |
+| JPEG           | :thumbsup:                        | :thumbsup:                    |                                     |
+| PNG            | :thumbsup:                        | :thumbsup:                    |                                     |
+| COG (local)    | :thumbsup:                        | :thumbsup:                    |                                     |
+| MBTiles        | :thumbsup:                        | :thumbsup:                    |  png, jpg compression               |
+| MBTiles        | :thumbsup:                        | :thumbsup:                    |  local vector tiles                 |
+| WM(T)S         | :thumbsup:                        | :thumbsup:                    |                                     |
+| XYZ tiles      | :thumbsup:                        | :thumbsup:                    | OpenStreetMap, Google Aerial, Bing. |
+| ECW            | :no_entry_sign:                   | :thumbsup:                    | Proprietary license                 |
+| MrSID          | :no_entry_sign:                   | :thumbsup:                    | Proprietary license                 |
+| COG (online)   | :no_entry_sign:                   | :thumbsup:                    |                                     |
```

Comment:
I would remove the column for QGIS - it does not add anything if there are thumbs up everywhere :)